packages/expo-cli/src/commands/apply/configureAndroidProjectAsync.ts

```diff
@@ -54,6 +55,7 @@ async function modifyMainActivityJavaAsync(
 
 export default async function configureAndroidProjectAsync(projectRoot: string) {
   const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
+  const username = await UserManager.getCurrentUsernameAsync();
```

Comment:
this is the first property that depends on the user being signed in. i don't want to block this pr on this, but we may want to think about the following:
- should still be possible to eject without signing in (this pr maintains that)
- let people know who they are signed in as / if they are not signed in / give an opportunity to sign in
- users will need to be signed in to download their build credentials if they keep them on our credentials service
- at some point we may need users to sign in their apple account or provide some other additional info eg: around entitlements like icloud